Datastore/src/Operation.php

```diff
@@ -497,6 +503,15 @@ public function runQuery(QueryInterface $query, array $options = [])
             if (isset($res['query'])) {
                 $runQueryObj = new Query($this->entityMapper, $res['query']);
             }
+            if (isset($res['query']['limit'])) {
+                $remainingLimit = $res['query']['limit'];
+            }
+            if (isset($remainingLimit['value'])) {
+                $remainingLimit = $remainingLimit['value'];
+            }
+            if (isset($remainingLimit)) {
```

Comment:
nit: I'd recommend being explicit and using `!is_null` here, since `isset` also applies to variables which haven't been defined